src/Zope2/utilities/tests/test_zconsole.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+import tempfile
+import unittest
+
+if sys.version_info.major == 2:
+    from StringIO import StringIO
```

Comment:
There is `six.StringIO` which does this, too, see https://pythonhosted.org/six/#six.StringIO